content/fast/minify-css/index.md

```diff
@@ -0,0 +1,162 @@
+---
+page_type: guide
+title: Minify CSS
+author: demianrenzulli
+description: |
+  Learn how to minify CSS files, to improve performance, without affecting how the browser process the styles.
+web_lighthouse:
+- render-blocking-resources
+web_updated_on: 
+web_published_on: 
+wf_blink_components: N/A
+---
+
+# Minify CSS
+
+CSS files can contain unnecessary characters, such as comments, whitespaces, and indentation. In production versions of the sites, these bytes can be safely removed, in order to reduce file sizes, without affecting how the browser processes the styles. We call this technique **minification**.
+
+## Loading unminified CSS
+
+Take a look at the following CSS block:
+
+```
+body {
+  font-family: "Benton Sans", "Helvetica Neue", helvetica, arial, sans-serif;
+  margin: 2em;
+}
+
+/* all titles need to have the same font, color and background */
+h1 {
+  font-style: italic;
+  color: #373fff;
+  background-color: #000000;
+}
+
+h2 {
+  font-style: italic;
+  color: #373fff;
+  background-color: #000000;
+}
+```
+
+This content is easy to read, at the cost of producing a larger than necessary file:
+
+- It uses spaces for indentation purposes and contains comments, which will be ignored by the browser.
+- The `<h1>` and `<h2>` elements have the same styles: instead of declaring them separately: "`h1 {...} h2 {...}`" they could be expressed as "`h1, h2{...}`".
+- The **background-color**, `#000000` could be expressed as just `#000`.
+
+After applying these observations, you would obtain a more compact version of the same styles:
+
+```
+body{font-family:"Benton Sans","Helvetica Neue",helvetica,arial,sans-serif;margin:2em}h1,h2{font-style:italic;color:#373fff;background-color:#000}
+```
+
+In practice, you would like to introduce this technique as one of the steps of your build process. In this guide, you’ll learn how to do it by using a popular build tool: [Webpack](https://webpack.js.org/).
+
+## Measure
+
+You’ll apply CSS minification to the [Fav Kitties site](https://minify-css-unoptimized.glitch.me/), which has been used for other web.dev articles as well.
+
+First, obtain the size of the CSS file used by the site:
+
+1. Open the [site](https://minify-css-unoptimized.glitch.me/) in Chrome.
+1. Press `Control+Shift+J` or `Cmd+Option+J` (Mac), to open DevTools.
+1. Click on the **Network** panel and filter for **CSS**.
+1. Make sure **Disable Cache** is checked and reload the page.
+
+<img class="screenshot" width="510px" height="96px" src="./cdt-css-unoptimized.png" alt="DevTools CSS unoptimized trace">
+
+The size of the file is **1.7KB**. If you now click on the file, and go to the **Response** tab, you’ll see that the content of the file includes things like indentation and whitespaces:
```

Comment:
I wonder if it's worth us adding in significantly more styling into this example so that the CSS file we're working with feels a little more real-world (e.g 10-40KB). I think the content of this guide completely makes sense, but we also want to avoid folks feeling like this is a micro-optimization (if we're shrinking that 1.7KB done), right?

Reply:
I can work on a new version of Fav Kittens to use more styles. 

An additional benefit is that, If we make them grow to 10-40KB, as suggested, we'll likely surpass [LH threshold](https://github.com/GoogleChrome/lighthouse/blob/69d312387141da484a98a610e9047466bbc2c362/lighthouse-core/audits/byte-efficiency/unminified-css.js#L24) of 2KB of savings, so that the "Minify CSS" LH suggestion can be shown at the beginning, and fixed at the end.

Also, we can reuse that demo for the "Using Media Queries" guide 👍

Reply:
Waiting on a new Fav Kittens version sgtm :)

Reply:
I'm rolling through some reviews today-- is this one to hold off on until Fav Kittens arrives?

Reply:
@Meggin sorry for the delayed response, just touched base with Addy, and we'll be working on a new version of the article, based on a slightly different Fav Kittens site. Hope to have an updated version to share soon.